src/shared/GenericSetHead.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Head from 'next/head';
+
+interface GenericSetHeadProps {
+  title: string;
+  metadata?: Array<{
+    property: 'keyword' | 'description' | 'image' | 'url';
```

Comment:
@madhav-devslane We should keep this as simple string as it can contain very dynamic values.